indexer/feature_visibility.cpp

```diff
@@ -223,12 +223,16 @@ namespace
 
   /// Add here all exception classificator types: needed for algorithms,
   /// but don't have drawing rules.
+  /// Warning: Geometry of features with always existing types will be indexed in mwm on all
+  /// zoom levels. If you add alaways existing type to drawing types, a displacement of icons will
```

Comment:
s/a displacement/the displacement/
or just "displacement"

Reply:
++ the